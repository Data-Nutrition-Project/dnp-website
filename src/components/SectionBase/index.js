import React from "react"
import styles from "./styles.module.css"
const SectionBase = () => {
  return (
    <div className={styles.sectionBase}>
      <h1 className={styles.sectionBaseHeader}></h1>
      <span className={styles.weightedLine}>
        <h2></h2>
      </span>
      <span className={styles.lightLine}>
        <h1></h1>
      </span>
      <span className={styles.lightLine}>
        <h1></h1>
      </span>
    </div>
  )
}

export default SectionBase
