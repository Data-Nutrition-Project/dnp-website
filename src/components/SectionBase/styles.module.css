@value layout: "../layout.css";

@value lightGray from layout;
@value borderDarkGray from layout;

.sectionBase {
  overflow: hidden;
  width: 77%;
  border-top: 1rem solid #000;
  border-right: solid 0.2rem #000;
  border-left: solid 0.2rem #000;
  border-bottom: solid 0.2rem #000;
  margin-top: 2.1875rem;
  padding: 4.6875rem 3.4375rem;
}

.blockDiv {
  display: block;
}

.hiddenDiv {
  display: none;
}
