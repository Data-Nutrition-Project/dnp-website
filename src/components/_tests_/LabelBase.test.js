import React from "react"
import renderer from "react-test-renderer"
import SectionBase from "../SectionBase"

describe("SectionBase", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SectionBase />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
