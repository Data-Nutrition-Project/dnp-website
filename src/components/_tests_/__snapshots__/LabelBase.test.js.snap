// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionBase renders correctly 1`] = `
<div
  className="sectionBase"
>
  <h1
    className="sectionBaseHeader"
  />
  <span
    className="weightedLine"
  >
    <h2 />
  </span>
  <span
    className="lightLine"
  >
    <h1 />
  </span>
  <span
    className="lightLine"
  >
    <h1 />
  </span>
</div>
`;
